src/components/Text/index.tsx

```diff
@@ -3,10 +3,8 @@ import { FC } from 'react'
 import { StyledText } from './typescript/text.style'
 import { IProps } from './typescript/text.types'
 
-const Text: FC<IProps> = (props) => {
+export const Text: FC<IProps> = (props) => {
```

Comment:
If we want to we can enforce this style with eslint-plugin-import rule `no-default-export`

Reply:
I will add this yep!

Reply:
https://github.com/benchttp/webapp/issues/37 for the ticket